_topics/en/anonymity-networks.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Anonymity networks
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - Tor
```

Comment:
maybe "onion" as well (in Bitcoin Core, use of "tor" is increasingly being replaced by "onion")

Reply:
I'm not sure about this given that LN uses "onion" for a bunch of things that aren't Tor related.  It might even be worth having a discussion in `#lightning-dev` about the Tor project's recommendation to call things onions versus LN's use of the same term for a bunch of stuff.  (Obviously both projects are using the term correctly, it's just conflationary.)

Reply:
I agree that linking here from the very broad "onion" word would be confusing for people expecting to read about BOLT 4, unless we included a paragraph that the word onion is overloaded.